templates/agent/keymanager/keymanager.go

```diff
@@ -115,13 +115,6 @@ func (p *Plugin) SignData(ctx context.Context, req *keymanagerv1.SignDataRequest
 	return nil, status.Error(codes.Unimplemented, "not implemented")
 }
 
-// SetLogger is called by the framework when the plugin is loaded and provides
-// the plugin with a logger wired up to SPIRE's logging facilities.
-// TODO: Remove if the plugin does not need the logger.
-func (p *Plugin) SetLogger(logger hclog.Logger) {
-	p.logger = logger
-}
-
 // BrokerHostServices is called by the framework when the plugin is loaded to
```

Comment:
We should keep BrokerHostServices grouped with SetLogger, since they are both in the same class of methods (called by the plugin framework)

Reply:
87a5532